src/migrations/20190906054214-create-users.js

```diff
@@ -0,0 +1,65 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('users', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    username: {
+      type: Sequelize.STRING
+    },
+    firstName: {
+      type: Sequelize.STRING
+    },
+    lastName: {
+      type: Sequelize.STRING
+    },
+    email: {
+      type: Sequelize.STRING
+    },
+    password: {
+      type: Sequelize.STRING
+    },
+    isVerified: {
+      type: Sequelize.BOOLEAN
+    },
+    roleId: {
+      type: Sequelize.INTEGER
```

Comment:
Is this suppose to be data type of interger ? @OlawaleJoseph

Reply:
yes